mvpa2/generators/permutation.py

```diff
@@ -50,11 +50,12 @@ def __init__(self, attr, count=1, limit=None, assure=False,
           all listed attributes.
         count : int
           Number of permutations to be yielded by .generate()
-        limit : None or str or dict
+        limit : None or str or list or dict
           If ``None`` all attribute values will be permuted. If an single
           attribute name is given, its unique values will be used to define
           chunks of data that are permuted individually (i.e. no attributed
-          values will be replaced across chunks). Finally, if a dictionary is
+          values will be replaced across chunks). If a list given, then combination
+          of those attributes is used as a pair. Finally, if a dictionary is
```

Comment:
Somewhat unclear to me. Do you mean "If a list with n elements, then all possible combinations of those attributes are used as n-tuples"?